src/cljs/rems/spa.cljs

```diff
@@ -330,10 +329,15 @@
 
 (defn page []
   (let [page-id @(rf/subscribe [:page])
-        grab-focus? @(rf/subscribe [::grab-focus?])]
+        grab-focus? @(rf/subscribe [::grab-focus?])
+        theme @(rf/subscribe [:theme])]
     (lazy-load-data!)
     [:div
      [nav/navigation-widget]
+     (when-not (:navbar-logo theme)
+       [nav/logo])
+     (when (= page-id :home)
+          [nav/logo])
```

Comment:
btw the logo can get duplicated because both of these conditions can be active at once

Reply:
that is true, I did notice it today 
<img width="1390" alt="Screenshot 2021-01-21 at 16 37 14" src="https://user-images.githubusercontent.com/7344718/105365535-0991ab00-5c07-11eb-9ac3-fa0f5a2f472c.png">
hahah, two is better than one :) ahah